scripts/mktcli.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env babel-node
+
+import { eth, Log, cli, contracts } from 'decentraland-commons'
+import { loadEnv } from './utils'
+import { decodeAssetId, encodeAssetId } from './monitor/utils'
+
+const log = new Log('mktcli')
```

Comment:
If we're having a _global_ cli, it seems the perfect place to put the rest of the scripts we're using independently. So for example we could do:

```bash
npm run cli monitor
npm run cli renew-blockchain
# etc
```

because if we don't we risk continuing to add more and more standalone scripts.

This can be a separate PR!, but what do you think? the alternative is to rename this file to something along the lines of `land-manager` maybe?